### PR TITLE
Adds a pattern draft for making governance levels explicit.

### PR DESCRIPTION
**Pull request by MaineC**
This adds a pattern for making several levels of openess wrt. governance levels
explicit to project contributors. Inspired by several versions of Open Source
project governance (e.g. single vendor with full control, foundation hosted with
vendor neutrality as a goal etc.) this pattern tries to translate these versions
into an InnerSource context.

### CODE REVIEW

File: README.md
Comment (spier) [inline README.md]:
```suggestion
* [Transparent Governance Levels](patterns/1-initial/governance-levels.md) - *There are projects in multiple stages of InnerSource adoption. Contributors get confused when working with projects that are at different stages.*
```

I would use the full title here.

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
Transparent Governance Levels
```

Title-casing this one like for most other patterns.

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
Initial (Early draft)
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
has a shared ownership model with [Trusted Committers](../2-structured/trusted-committer.md) from multiple teams.
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
* **Bug reports and issues welcome**: People outside the core development team are
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
* **Contributions welcome**: People outside the core development team may use the
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
* **Shared write access**: In addition to the above people outside the core
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]:
```suggestion
* **Shared ownership**: Members of different teams collaborate on the project as
```

File: patterns/1-initial/governance-levels.md
Comment (spier) [inline patterns/1-initial/governance-levels.md]: Is "core development team" here the same as "backing team" which you used above?

File: README.md
Comment (spier) [inline README.md]: Also this bullet should be moved to the group **Maturity Level 1: Initial**, right below the **Reluctance to Receive Contributions** pattern.

Sorry, I know that the list of patterns is confusing right now. We are working on cleaning this up.

**Review by spier — CHANGES_REQUESTED** [review]
Great first version @MaineC! 
I would be happy to merge that in as a level 1 (Initial) pattern.

I sent some inline suggestions for you to review.

Update:
I just noticed that a **Patlet** is missing. Maybe you confused **Patlet** and **Context**?

**Comment by spier** [comment]
Also I committed and resolved some of my own suggestions that were only required to make our Markdownlint check happy :)

File: patterns/1-initial/governance-levels.md
Comment (MaineC) [reply patterns/1-initial/governance-levels.md]: I think I'm lacking a good term here: What I'm trying to explain is that there is a difference between a project that is carried forward by individuals working under one line manager, delivering one roadmap vs. a project where individuals from multiple such teams collaborate.

Comment (spier) [reply patterns/1-initial/governance-levels.md]: I see. I don't know the best term here either.
For the time being I would then suggest to just introduce and define **a term**, likely towards the beginning of the Pattern, and then stick to that term through out the pattern.

As we improve this pattern over time we might come up with another term, possibly influenced by other patterns as well. We may even want to pull such terms that we attribute certain meaning to into a central glossary to help pattern authors and readers to use these terms in a similar manner.

File: patterns/1-initial/governance-levels.md
Comment (spier) [reply patterns/1-initial/governance-levels.md]: Using the term "initial" according to the maturity model.

File: patterns/1-initial/governance-levels.md
Comment (spier) [reply patterns/1-initial/governance-levels.md]: Just formatting, so I suspect these will be ok :)

Comment (spier) [reply patterns/1-initial/governance-levels.md]: Will merge these now.

**Comment by spier** [comment]
@MaineC I took the liberty to resolve/merge some of my suggested edits that I did not consider controversial. Hope I didn't overstep there, so feel free the change them back if you feel strongly about any of them.

**Comment by MaineC** [comment]
Oh - that's what caused those pesky merge conflicts when I tried to push the changes I made according to your review yesterday :/

**Comment by MaineC** [comment]
Are there any more open suggestions from your side that I should address?

**Comment by MaineC** [comment]
One more caveat: Please leave this pull request open for a few more days, I've asked a few people via Twitter whether they'd have time to add their perspective. 

Edit: One piece of background to integrate here: https://twitter.com/galoppini/status/1352270671011835905

Edit 2: One very current case where we could draw inspiration from here: https://twitter.com/MaineC/status/1352123851237429248

Edit 3: https://twitter.com/MaineC/status/1351913663020617732 ... this is the origin of this pull request for comparison

Any help with integrating the different perspectives welcome. :)

File: README.md
Comment (spier) [inline README.md]: This bullet should be moved to the group **Maturity Level 1: Initial**, right below the **Reluctance to Receive Contributions** pattern.

Sorry, I know that the list of patterns is confusing right now. We are working on cleaning this up.

**Comment by spier** [comment]
@MaineC only two more suggestions from my end. See:
https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/281/files

And great job bringing in more people into the conversation!

File: README.md
Comment (MaineC) [reply README.md]: Feel free to move that as well, I believe the branch in the fork for this pull request should be writable for maintainers of the original project.

**Comment by MaineC** [comment]
Side note: After going through the paper linked above I believe we can come up with a few more fine-grained patterns along the lines of "If you want to increase contributor engagement, try making your InnerSource project more transparent and more accessible".

File: README.md
Comment (spier) [reply README.md]: Done

**Review by spier — APPROVED** [review]
LGTM!

**Comment by spier** [comment]
> One more caveat: Please leave this pull request open for a few more days, I've asked a few people via Twitter whether they'd have time to add their perspective.
> 
> Edit: One piece of background to integrate here: https://twitter.com/galoppini/status/1352270671011835905
> 
> Edit 2: One very current case where we could draw inspiration from here: https://twitter.com/MaineC/status/1352123851237429248
> 
> Edit 3: https://twitter.com/MaineC/status/1351913663020617732 ... this is the origin of this pull request for comparison
> 
> Any help with integrating the different perspectives welcome. :)

@MaineC this PR looks great to me now. Do you still want to keep it open, or shall I merge it?

**Comment by spier** [comment]
@MaineC shall we merge this PR, even if only to get the pattern "out there" in it's initial form and make it easier to link to it?

Also do you want to add yourself to the pattern in a `## Author` section?

**Comment by MaineC** [comment]
Feel free to merge the pattern. I think we can add additional detail as separate patterns.

**Comment by spier** [comment]
@MaineC excellent. Merging now.

**Pull request merged**
